### ISSUE TITLE
Generate initial solution for cold start

### ISSUE THREAD

**Issue by PatrickHenkel** [RWTH-EBC/DDMPC#13]
Using neural networks, the gradient can vanish/explode when evaluated far from the training data regime. To avoid vanishing/exploding gradients during the cold start of the mpc (initialized with 0), it is useful to generate a simple initial guess which is close to the training data regime
